roles/internal/grok/vars/main.yml

```diff
@@ -0,0 +1,4 @@
+grok_clone_directory: /opt/grok
+grok_version_tag: v2.3.0
+
+cantaloupe_OpenJpegProcessor_path_to_binaries: /usr/local/bin
```

Comment:
This shouldn't be defined in this playbook for modularity. You want this role to stand on its own not to be setting variables from other roles. I would set this in this with the rest of the cantaloupe variables in: 
https://github.com/Islandora-Devops/claw-playbook/blob/master/inventory/vagrant/group_vars/tomcat.yml#L37-L53